src/prx/helpers.py

```diff
@@ -276,97 +277,136 @@ def compute_relativistic_clock_effect(sat_pos_m: np.array, sat_vel_mps: np.array
 
     return relativistic_clock_effect_m
 
-
-def parse_rinex_nav_file(file_path):
+def parse_rinex_nav_file(file_paths):
```

Comment:
`A2` should be renamed `T`
`T` should be renamed `W`

Reply:
I have made the desired changes and pushed it